contracts/src/v0.8/dev/ArbitrumValidator.sol

```diff
@@ -0,0 +1,211 @@
+ */
+contract ArbitrumValidator is SimpleWriteAccessController, AggregatorValidatorInterface {
+  // Follows: https://eips.ethereum.org/EIPS/eip-1967
+  address constant private s_arbitrumFlag = address(bytes20(bytes32(uint256(keccak256("chainlink.flags.arbitrum-offline")) - 1)));
+  bytes constant private s_raiseFlagData = abi.encodeWithSelector(FlagsDevInterface.raiseFlag.selector, s_arbitrumFlag);
+  bytes constant private s_lowerFlagData = abi.encodeWithSelector(FlagsDevInterface.lowerFlag.selector, s_arbitrumFlag);
+  uint32 constant private s_L2GasLimit = 30000000;
+
+  address private s_flagsAddress;
+  ArbitrumInboxInterface private s_arbitrumInbox;
+  SimpleWriteAccessController private s_gasConfigAccessController;
+
+  struct GasConfiguration {
+    uint256 maxSubmissionCost;
+    uint32 maximumGasPrice;
+    uint256 gasCostL2;
+    address refundableAddress;
+  }
+  GasConfiguration private s_gasConfig;
+
+  /**
+   * @notice emitted when a new gas configuration is set
+   * @param maxSubmissionCost maximum cost willing to pay on L2
+   * @param maximumGasPrice maximum gas price to pay on L2
+   * @param gasCostL2 value to send to L2 to cover gas fee
+   * @param refundableAddress address where gas excess on L2 will be sent
+   */
+  event GasConfigurationSet(
+    uint256 maxSubmissionCost,
+    uint32 maximumGasPrice,
+    uint256 gasCostL2,
+    address refundableAddress
+  );
+
+  /**
+   * @notice emitted when a new gas access-control contract is set
+   * @param old the address prior to the current setting
+   * @param current the address of the new access-control contract
+   */
+  event GasAccessControllerSet(
+    address old, 
+    address current
+  );
+
+  /**
+   * @param inboxAddress address of the Arbitrum Inbox L1 contract
+   * @param flagAddress address of the Chainlink L2 Flags contract
+   * @param gasConfigAccessController address of the access controller for managing gas price on Arbitrum
+   * @param maxSubmissionCost maximum cost willing to pay on L2
+   * @param maximumGasPrice maximum gas price to pay on L2
+   * @param gasCostL2 value to send to L2 to cover gas fee
+   * @param refundableAddress address where gas excess on L2 will be sent
+   */
+  constructor(
+    address inboxAddress,
+    address flagAddress,
+    address gasConfigAccessController,
+    uint256 maxSubmissionCost,
+    uint32 maximumGasPrice,
+    uint256 gasCostL2,
+    address refundableAddress
+  ) {
+    require(flagAddress != address(0), "Invalid Flags contract address");
+    s_arbitrumInbox = ArbitrumInboxInterface(inboxAddress);
+    s_gasConfigAccessController = SimpleWriteAccessController(gasConfigAccessController);
+    s_flagsAddress = flagAddress;
+    setGasConfigurationInternal(maxSubmissionCost, maximumGasPrice, gasCostL2, refundableAddress);
+  }
+  
+  /**
+   * @notice makes this contract payable. It need funds in order to pay for L2 transactions fees
+   */
+  fallback() external payable {}
```

Comment:
We shouldn't use the fallback function as we are not handling fallback scenarios but just receiving funds. A `receive` function is more explicit with less surface area.